app/assets/javascripts/application.js

```diff
@@ -4,6 +4,7 @@
 //= require govuk_publishing_components/components/checkboxes
 //= require govuk_publishing_components/components/copy-to-clipboard
 //= require govuk_publishing_components/components/govspeak
+//= require govuk_publishing_components/components/select-with-search
```

Comment:
is it worth including some info or a link to explain what behaviour this JS library adds? (e.g. I have no idea!)